src/main/resources/edu/cornell/kfs/fp/businessobject/datadictionary/InternalBillingItem.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:p="http://www.springframework.org/schema/p"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+                http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+    <!--
+        To properly allow the Data Dictionary to interpret the "exclusiveMin" property as unset,
+        we must set it to null instead of an empty string.
+     -->
+
+    <bean id="InternalBillingItem-itemUnitAmount" parent="InternalBillingItem-itemUnitAmount-parentBean"
+            p:exclusiveMin="#{null}"/>
```

Comment:
I haven't seen exclusiveMin in DD before.  What does it do (I'd like to understand)?

Reply:
"exclusiveMin" is a constraint that can be added for DD-based field validation. It validates that a field's value is greater than the given value, though it only works for numeric or date values, or for strings formatted as numeric or date values. (There's also a related "inclusiveMax" constraint for restricting the upper bound.)

Reply:
Thanks for the explanation.  Why did we need to add it to itemUnitAmount, since it doesn't look like we are applying different validation to it.

Reply:
The "exclusiveMin" validation is independent of the validation pattern replacement that also needed to be made on one of the properties. In base code, both of the affected item fields had "exclusiveMin" values of 0, thus preventing users from entering negative values. That was why the "exclusiveMin" validation needed to be cleared out on the two fields.

Reply:
Thanks for the clarification